### DISCUSSION TITLE
Display TUI over the console

### DISCUSSION THREAD

**Discussion by jdhitsolutions** [tui-cs/Terminal.Gui#3616] [General]
I can create a sized TUI-based window that does everything I need. 

```powershell
$win = [Terminal.Gui.Window] @{
        Title  = 'Get-TuiCredential'
        Width  = 45
        Height = 9
        X      = [Terminal.Gui.Pos]::Center()
        Y      = [Terminal.Gui.Pos]::Center()
    }
```

However, whenever I run it, the console is completely covered.

![image](https://github.com/user-attachments/assets/3a9185cf-4850-4633-a26a-10645e816bbc)

Is there a way to display the window over the console without covering it completely? I'm running this in a PowerShell window in Windows Terminal.

**Comment by BDisp** [comment]
For now it isn't possible. There is already an open issue #272 asking for that.

**Reply by jdhitsolutions** [reply]
Thanks for the confirmation. I suspected as much.
